Assignment/Program.cs

```diff
@@ -1,7 +1,18 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿while (true) {
+   Console.Write ("Enter number : ");
+   if (int.TryParse (Console.ReadLine (), out int num) && num > 0) {

```

Comment:
Same mistake string input could be null. Please check.

Reply:
Nullability is handled while passing "Console.ReadLine" as a parameter inside TryParse.
Other values are also handled.